modules/Kconfig.cmsis

```diff
@@ -28,3 +28,13 @@ menuconfig CMSIS_DSP
 if CMSIS_DSP
 source "modules/Kconfig.cmsis_dsp"
 endif
+
+menuconfig CMSIS_NN
+	bool "CMSIS-NN Library Support"
+	depends on CPU_CORTEX
+	depends on NEWLIB_LIBC
+	select CMSIS_DSP
```

Comment:
Note that simply selecting `CMSIS_DSP` symbol may not necessarily select the CMSIS-DSP components that CMSIS-NN depends on. This should resolve the exact CMSIS-DSP component dependec{y,ies}.

Reply:
Haven't I done that already starting from line line 37?

Reply:
I was looking at the first commit, which did not have these.